.github/workflows/nuget-preview.yml

```diff
@@ -0,0 +1,69 @@
+name: GitHub Actions
+
+on: [ pull_request ] # testing only
+#on:
+#  schedule:
+#  - cron: "5 4 3 */1 *" # once a month, at 4:05 on 3rd
+
+jobs:
+  release:
+    name: Release Preview
+    runs-on: ubuntu-latest
+    environment: NuGet
+    timeout-minutes: 10
+
+    env:
+      PYTHONNET_SHUTDOWN_MODE: Normal
+
+    steps:
+      - name: Get Date
+        run: |
+          echo "DATE_VER=$(date "+%y-%m-%d")" >> $GITHUB_ENV
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Setup .NET
+        uses: actions/setup-dotnet@v1
+
+      - name: Set up Python 3.8
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+          architecture: x64
+
+      - name: Install dependencies
+        run: |
+          pip install --upgrade -r requirements.txt
+
+      - name: Build and Install
+        run: |
+          python setup.py configure
+          pip install -v .
+
+      - name: Python Tests
+        run: pytest
+        env:
+          PYTHONNET_PYDLL: libpython3.8.so
+
+      - name: Embedding tests
+        run: dotnet test --runtime any-ubuntu src/embed_tests/
+        env:
+          PYTHONNET_PYDLL: libpython3.8.so
+
+      - name: Python tests run from .NET
+        run: dotnet test --runtime any-ubuntu src/python_tests_runner/
+        if: ${{ matrix.os == 'windows' }} # Not working for others right now
+        env:
+          PYTHONNET_PYDLL: libpython3.8.so
+
+      - name: Pack
+        run: dotnet pack --configuration Release --version-suffix preview${{env.DATE_VER}}
+
+      - name: Publish NuGet
+        run: dotnet nuget push --api-key ${NUGET_KEY}
+        env:
+          NUGET_KEY: ${{ secrets.NUGET_MONTHLY }}
```

Comment:
I think this way you'll leak the key.

Reply:
Without this Visual Studio tends to add UTF-8 BOM header to solution, leading to PRs with the entire file replaced. It simply ignores `utf-8`.

Reply:
I don't think so, but the reason I had to try to add this turned out to be bogus: it was not picked up, because PR was from my fork. I moved the PR branch into this repository instead, now it is getting picked up correctly. See #1381